script/Ciel/Day 14/14_01A_CIEL14A_2.txt

```diff
@@ -505,7 +549,7 @@ Quickly, before she removes the stake!"
 [sha:eb9d5d1d2831d52cc78db1578a1c8e94e9d7f40e]{
 -- Page 47, Offset 37749.
 -- 当惑する俺の腕を掴んで、先輩は強引に走り出した。
-While I'm still perplexed, Senpai grabs on to my arm and forces me to run.
+My mind is still reeling, but Senpai grabs me by the arm and drags me after her.
 }
 [sha:bce4ae1092cf65f29db7f66196a9a08d406dfeae]{
 -- Page 48, Offset 37750.
```

Comment:
He's asking how.
also some punctuation.
>Wait, Senpai, how'd you——

Reply:
I don't think how is quite right, it makes it sound like it's confused about how she could get there rather than why she's there. I feel like 'what' might be better in full - "what [are you doing back so soon]" -> she parries it to "what [the fuck is going on here I left you alone for five minutes]"

Reply:
maybe just expand both to "what's going on"